transport/inbound.go

```diff
@@ -46,3 +46,17 @@ type Inbound interface {
 	// Implementations can assume that this function is called at most once.
 	Start(handler Handler, deps Deps) error
 }
+
+// OnewayInbound is a transport that knows how to receive oneway requests
+// for procedure calls.
+type OnewayInbound interface {
+	BaseInbound
+
+	// Starts accepting new requests and dispatches them to the given Handler.
+	//
+	// The function MUST return immediately, although it SHOULD block until
+	// the inbound is ready to start accepting new requests.
+	//
+	// Implementations can assume that this function is called at most once.
+	Start(handler OnewayHandler, deps Deps) error
```

Comment:
So the "handler" in this case will be the dispatcher or one of the inbound handlers.  Is the plan to wire this so the registry also keeps track of "OnewayHandlers" as separate inbounds/outbounds?